website/app/components/doc/image-caption/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class DocImageCaptionComponent extends Component {
+  constructor() {
```

Comment:
We don't need a constructor here

Reply:
Good point. Will follow this pattern going forward.